src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureImageOperationUtilTest.java

```diff
@@ -1,13 +1,35 @@
 package cloud.fogbow.ras.core.plugins.interoperability.azure.util;
 
+import cloud.fogbow.common.exceptions.UnexpectedException;
 import cloud.fogbow.ras.api.http.response.ImageSummary;
 import cloud.fogbow.ras.core.plugins.interoperability.azure.compute.sdk.model.AzureGetImageRef;
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
 import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+        AzureImageOperationUtil.class,
+        URLDecoder.class,
+        URLEncoder.class
+})
 public class AzureImageOperationUtilTest {
```

Comment:
You could add a method that tests the encode and decode method in the same test case.

Reply:
Done.

Reply:
Done.